docs/api-client/README.md

```diff
@@ -1530,6 +1530,50 @@ Example
 "signature + userId/contractId pair stored"
 ```
 
+### prepareSignature
+
+Method used in connection with a contract signatory using own [MetaMask](https://metamask.io/) or similar account to record an electronic signature and a contract ID, which is a cryptographic hash of the contract, on a blockchain network. Upon completion of that transaction for signing the contract and once the transaction hash has been sent back to the server for validation and record-keeping, the address, signature, and network associated with the transaction are also sent back.
```

Comment:
I recently updated the lead sentence in the `sendTxHash` method to this:
"Method used in connection with a contract signatory using own Ethereum account (including via [MetaMask](https://metamask.io/)) to record an electronic signature and a contract ID...."

Can use here too if you want but also fine to keep as is.

Might be good to clarify that "the address" is "the address of the smart contract that handles the signature transaction" we used that language for the `getCurrentNetwork` response description.